azerothcore/azerothcore-wotlk
# fix(db/creature): Replace Smart Script AI on Verog the Dervish so that he paths down h…

## Changes Proposed:
-  Verog the Dervish doesnt move when he spawns this is not as per videos available of this quest. He is supposed to RP walk down the hill. I think this lack of movement is contributing to the impression that he doesnt spawn. This PR makes him walk down the hill.

## Issues Addressed:
- Closes 
https://github.com/chromiecraft/chromiecraft/issues/114
and
https://github.com/azerothcore/azerothcore-wotlk/issues/11236

## SOURCE:
https://classic.wowhead.com/npc=3395/verog-the-dervish#comments Lots of peeps from 2006 and 2007 qqing that it takes lots of kills (30+) to get a spawn even as rescent as 2020.

## Tests Performed:
- I killed for about 30 minutes with only one spawn but was to far away to get to him in 60 seconds.
- I did finally get him to spawn while being close enough to see. He does spawn. 
- Vorag now spawns. Spends 25 secs walking down the Hill and 35 seconds waiting before he despawns. This is the same 60 seconds hes always done just with the missing RP walk added. 

## How to Test the Changes:
1. .go xyz -1208.59	-2776.1	93.1576
2. Start killing everything in site for 30 to 45 minutes.
3. When something yells
4. .go xyz -1208.59	-2776.1	93.1576
5. verify Vorag is walking toward you and doesn't disappear when he reaches the bottum of the ramp for at least 30 seconds. 


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
